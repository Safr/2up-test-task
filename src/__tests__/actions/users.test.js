import { addUsersSuccess, addUsersFailure } from '../../actions/users';

describe('Users Actions', () => {
  const user = {
    id: 1,
    name: 'name',
  };
  test('should add users', () => {
    const action = addUsersSuccess([user]);
    expect(action).toMatchSnapshot();
  });
  test('should add error', () => {
    const action = addUsersFailure('error');
    expect(action).toMatchSnapshot();
  });
});
