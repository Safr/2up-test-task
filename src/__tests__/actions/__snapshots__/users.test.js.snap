// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Users Actions should add error 1`] = `
Object {
  "error": "error",
  "type": "ADD_USERS_FAILURE",
}
`;

exports[`Users Actions should add users 1`] = `
Object {
  "type": "ADD_USERS_SUCCESS",
  "users": Array [
    Object {
      "id": 1,
      "name": "name",
    },
  ],
}
`;
